R/qenv-replace_code.R

```diff
@@ -0,0 +1,53 @@
+#' Replace code
+#'
+#' Replace last element of code stored in `qenv`.
+#'
+#' Replace the last element of the character vector stored in `@code` with provided `code`.
+#' This can be used to mask sensitive information, e.g. credentials.
```

Comment:
When someone from the outside will be looking at this then this functionality might be super suspicious. Can you please provide few words about the context - i.e. hiding tokens, passwords etc? I know that you have this already in vignette but that someone will be probably looking only here.

Reply:
That's what worries me, though. Our intentions are rather pure and we can dress it up in pretty examples but the fact is you can put whatever you want in there.

Also, doesn't "credentials" cover what you mean?

Reply:
> Also, doesn't "credentials" cover what you mean?

Yes